src/app/nation/nation.component.ts

```diff
@@ -71,7 +73,12 @@ export class NationComponent implements OnInit, AfterViewInit {
       .then((data) => {
         // _all_docs returns object with rows array of objects with 'doc' property that has an object with the data.
         // Map over data.rows to remove the 'doc' property layer
-        this.nations.data = data.rows.map(nation => nation.doc);
+        this.nations.data = data.rows.reduce((nations: any[], nation: any) => {

```

Comment:
My mistake earlier, this is not equivalent to only map, but I think using map & filter will be faster & simpler than reduce:

https://jsperf.com/map-filter-vs-reduce-simple